### ISSUE TITLE
maintainer addresses with non-standard characters

### ISSUE THREAD

**Issue by bbolker** [fmichonneau/foghorn#7]
**tl;dr** the maintainer-e-mail-to-URL conversion should also have a `gsub()` step substituting underscores for special characters ...

The [CRAN checks-by-maintainer page](https://cran.r-project.org/web/checks/check_summary_by_maintainer.html) says:

 > Maintainers can directly adress [sic] their results via https://CRAN.R-project.org/web/checks/check_summary_by_maintainer.html#address:id, where id is obtained from the shown email address with all characters different from letters, digits, hyphens, underscores, colons, and periods replaced by underscores.

(Let's hope there aren't two CRAN maintainers with e-mail addresses disambiguated *only* by special characters ...)

This is a little misleading or incomplete: the form shown above takes you to an *anchor* on the full CRAN form, whereas the form https://cran.r-project.org/web/checks/check_results_address.html takes you to a maintainer-specific page (also, typo in "adress" above ... I know these are CRAN's problems, not yours ...)

The maintainer address for `lme4` is "bbolker+lme4 at gmail.com".  This means I can get there via https://cran.r-project.org/web/checks/check_results_bbolker_lme4_at_gmail.com.html ... 

(I got very confused while working on  this issue, I think I was just trying a whole bunch of slightly wrong things.)

**Comment by fmichonneau** [comment]
I think this can be closed now?
